### PR TITLE
Bump version for 0.2.0 release

### PR DESCRIPTION
**Pull request by richardliaw**
Bump to 0.2.0!

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```
Comment (richardliaw) [inline setup.cfg:2]:
```suggestion
description-file = README.md

```

File: setup.py [hunk 2]
```diff
@@ -1,12 +1,23 @@
-from setuptools import setup
+import io
+import os
+from setuptools import setup, find_packages
+from tune_sklearn import __version__
+
+ROOT_DIR = os.path.dirname(__file__)
+
+VERSION = os.environ.get("TSK_RELEASE_VERSION", __version__)
```
Comment (krfricke) [inline setup.py:8]: Why are we checking for this environment variable?

**Review by krfricke — APPROVED** [review]
I just have a question regarding the env variable.

**Comment by richardliaw** [comment]
So that it is easy to do release candidates :)

File: setup.py (hunk 2 above)
Comment (richardliaw) [reply setup.py:8]: makes it easier to do release candidates / dryruns :)

**Pull request merged**